packages/desktop/src/lib.rs

```diff
@@ -174,7 +174,25 @@ pub fn launch_with_props<P: 'static + Send>(
                 }
 
                 if cfg!(debug_assertions) {
+                    // in debug, we are okay with the reload menu showing and dev tool
```

Comment:
we will keep menu in debug mode ?

Reply:
We want to provide the `inspect element` tool in debug which means we need to keep the menu around. So, yes.

Reply:
oh!